packages/xod-client-electron/src/view/actions.js

```diff
@@ -150,6 +156,37 @@ export const saveAll = createAsyncAction({
     complete: MESSAGES.SAVE_ALL_SUCCEED,
     error: MESSAGES.SAVE_ALL_FAILED,
   },
+  onComplete: (payload, dispatch, getState) => {
+    if (payload.updateProjectPath) {
```

Comment:
Better use early exit to remove one level of nesting:

```js
if (!payload.updateProjectPath) {
  return;
}
```

It took 5 mins for me to understand how this works because the name is misleading a bit. It would be more intuitive if named `shouldUpdateProjectPath`.